sickbeard/providers/zooqle.py

```diff
@@ -0,0 +1,152 @@
+# coding=utf-8
+# Author: medariox <dariox@gmx.com>
+#
+# This file is part of Medusa.
+#
+# Medusa is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Medusa is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Medusa. If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from requests.compat import urljoin
+
+from sickbeard import logger, tvcache
+from sickbeard.bs4_parser import BS4Parser
+
+from sickrage.helper.common import convert_size, try_int
+from sickrage.providers.torrent.TorrentProvider import TorrentProvider
+
+
+class ZooqleProvider(TorrentProvider):  # pylint: disable=too-many-instance-attributes
+    """Zooqle Torrent provider"""
+    def __init__(self):
+
+        # Provider Init
+        TorrentProvider.__init__(self, 'Zooqle')
+
+        # Credentials
+        self.public = True
+
+        # Torrent Stats
+        self.minseed = None
+        self.minleech = None
+
+        # URLs
+        self.url = 'https://zooqle.com/'
+        self.urls = {
+            'search': urljoin(self.url, '/search'),
+        }
+
+        # Proper Strings
+        self.proper_strings = ['PROPER', 'REPACK', 'REAL']
+
+        # Cache
+        self.cache = tvcache.TVCache(self, min_time=15)
+
+    def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-many-locals, too-many-branches
+        """
+        Zooqle search and parsing
+
+        :param search_string: A dict with mode (key) and the search value (value)
+        :param age: Not used
+        :param ep_obj: Not used
+        :returns: A list of search results (structure)
+        """
+        results = []
+
+        # Search Params
+        search_params = {
+            'q': '* category:TV',
+            's': 'dt',
```

Comment:
This is a small workaround, as the provider doesn't allow an empty search. * will search for anything and category:TV is needed to restrict the results to TV content. The other params are default for date sorting.
The full query looks like this: ```https://zooqle.com/search?q=%2A%20category%3ATV&s=dt&v=t&sd=d```